Testing/SystemTests/tests/qt/SliceViewerIntegrationTest.py

```diff
@@ -488,6 +488,30 @@ def runTest(self):
         pres.view.close()
 
 
+class SliceViewerTestAxesLimitsWithLogMatrixTransform(systemtesting.MantidSystemTest, HelperTestingClass):
+    def runTest(self):
+        HelperTestingClass.__init__(self)
+        limits = (-10.0, 10.0, -9.0, 9.0)
+        ws_nonrotho = CreateMDWorkspace(Dimensions=3,
+                                        Extents=','.join([str(lim) for lim in limits]) + ',-8,8',
+                                        Names='A,B,C',
+                                        Units='r.l.u.,r.l.u.,r.l.u.',
+                                        Frames='HKL,HKL,HKL')
+        expt_info_nonortho = CreateSampleWorkspace()
+        ws_nonrotho.addExperimentInfo(expt_info_nonortho)
+        SetUB(ws_nonrotho, 1, 1, 2, 90, 90, 120)
+        W_MATRIX = [1,1,0,-1,1,0,0,0,1]
+        ws_nonrotho.getExperimentInfo(0).run().addProperty('W_MATRIX', W_MATRIX, True)
+        pres = SliceViewer(ws_nonrotho)
+
+        # set nonorthog view and retrieve new limits
+        pres.nonorthogonal_axes(True)
```

Comment:
What is the purpose of this call to `nonorthogonal_axes` - is it to check there's no exception? It doesn't change the `proj_matrix` - so as long as we verify the `proj_matrix` is valid then it is probably a test for the presenter to check `non_orthogonal_axes` works with a valid `proj_matrix` (which I believe exists)?

Reply:
Ah yes good spot!